tests/unit/amo/components/TestAddon.js

```diff
@@ -879,7 +879,7 @@ describe(__filename, () => {
     expect(image.type()).toEqual('img');
     expect(image).toHaveClassName('Addon-theme-header-image');
     expect(image.prop('src')).toEqual(
-      'https://addons.cdn.mozilla.net/123/image.png',
+      'https://addons.cdn.mozilla.net/7123/image.png',
```

Comment:
This change was made because we now expect `thumbnail_url` to be used rather than `image_url`, correct? If so, we shouldn't be relying on default values and should be setting a specific value for the `addon` in the test during setup. I've commented on this a bit more at length in a comment about `TestSearchResult.js`.

Reply:
ahh...ok i'll take another look at this. thanks @bobsilverberg